install/0000_90_machine-config-operator_01_prometheus-rules.yaml

```diff
@@ -67,7 +67,7 @@ spec:
       rules:
         - alert: MCDRebootError
           expr: |
-            max by(namespace, node, pod) (increase(mcd_reboots_failed_total[15m])) > 0
+            mcd_reboots_failed_total > 0
```

Comment:
I'm a bit of an alert newbie, when you say `This will clear the alert immediately on a successful reboot`, what is processing that? I see we do a `Inc()` on the alert which is what I assume is keeping this active, but what clears it if we reboot after say, 5 failures? Is it purely time based?

Reply:
counter metrics get reset only when server gets restarted. In our case, successful reboot will cause MCD metrics running on the node set to 0 . https://prometheus.io/docs/instrumenting/writing_clientlibs/#counter .

Reply:
yep, as sinny said the counters get reset to 0, so that should clear the error. I think increase would've worked just as well, but seems a bit overkill since we don't need to look for transients/jumps but rather anything non zero